conf/submarine-site.xml

```diff
@@ -143,6 +143,12 @@
     <description>RuntimeFactory for Submarine jobs</description>
   </property>
 
+  <property>
+    <name>submarine.server.remote.execution.enabled</name>
```

Comment:
Amended to `submarine.server.prc.enabled`, will it be clearer?

Reply:
Done